docker/couchdb/monitor.sh

```diff
@@ -2,13 +2,13 @@
 
 # start couchdb exporter
 COUCH_CONFIG=/data/srv/current/auth/couchdb/couchdb_config.ini
-export COUCHDB_NODE_NAME="couchdb@$(hostname)"
 # test if file is not zero size
 if [ -s "${COUCH_CONFIG}" ]; then
-    sudo cp /etc/secrets/$fname /data/srv/current/auth/$srv/$fname
-    sudo chown $USER.$USER /data/srv/current/auth/$srv/$fname
-    nohup couchdb-prometheus-exporter -telemetry.address=":9984" -logtostderr=true \
-      --config=$COUCH_CONFIG -databases.views=false --databases=_all_dbs 2>&1 1>& couchdb_exporter.log < /dev/null &
+    #sudo cp /etc/secrets/$fname /data/srv/current/auth/$srv/$fname
+    #sudo chown $USER.$USER /data/srv/current/auth/$srv/$fname
```

Comment:
Why secrets are removed? I think it is needed to be able to query CouchDB and collect the metrics.